packages/uxpin-merge-cli/src/common/services/UXPin/DeleteTag.ts

```diff
@@ -0,0 +1,35 @@
+import { isTestEnv } from '../../../program/env/isTestEnv';
+import { requestPromiseWithEnhancedError } from '../../../utils/requestPromiseWithEnhancedError';
+import { getAuthHeaders } from './headers/getAuthHeaders';
+import { getUserAgentHeaders } from './headers/getUserAgentHeaders';
+
+export const enum RepositoryPointerType {
```

Comment:
it could go to separate file because its duplicated

Reply:
For this `RepositoryPointerType` enum should I remove it from all the files it is being duplicated in? For example `DeleteRepositoryPointerToBranch` `DeleteTag` `updateRepositoryPointerToBranch` `GetRepositoryPointerDefaultHandler` Place it in its own file then import it to each of these files instead? If so, where is a good place to keep our enums or Types in this repo?